src/form/src/components/styled/StyledForm.js

```diff
@@ -91,9 +91,9 @@ const StyledForm = styled(Form)`
   }
 
   .grid-container {
-    display: flex;
+    display: grid;
     flex: 1;
-    flex-direction: column;
+    grid-template-columns: auto min-content;
```

Comment:
Why are there 2 columns? 

`flex-wrap` defaults to `nowrap`, so in theory, wrapping should behave the same way, i.e. no wrapping. However, we need to confirm because maybe `flex-wrap` has been changed somewhere else. Have you checked how this change impacts wrapping?

Reply:
The second column is for the edit button.

I really only used `flex: 1` to get it to stretch across the entire container. I could replace this with `width: 100%` if you're worried.

Reply:
I don't think `width: 100%` is necessary.